cypress/fixtures/stubbing/default/workspaces.js

```diff
@@ -19,8 +29,95 @@ export const WORKSPACE_EXAMPLE = {
   sendInputToDataWarehouse: true,
   useDedicatedEventHubNamespace: true,
   sendScenarioMetadataToEventHub: true,
-  security: { default: 'admin', accessControlList: [] },
+  security: { default: ROLES.SCENARIO.ADMIN, accessControlList: usersAccess },
 };
 
 export const DEFAULT_WORKSPACE = WORKSPACE_EXAMPLE;
```

Comment:
Isn't it redundant to assign an existing const to a new const ? :thinking:

Reply:
It was already like that. I don't modify it to avoid breaking some things.

Reply:
You are right ! thanks :pray: